### ISSUE TITLE
Batching in 3.x

### ISSUE THREAD

**Issue by aliksend** [Akryum/vue-apollo#246]
Hi

I'm now on vue-apollo 3.0.0-beta.4. I tried to use [batching](https://www.apollographql.com/docs/react/advanced/network-layer.html#query-batching), convinced that multiple queries combines to one request to backend , but vue-apollo can't read data. In result callback I got this: 
```js
{data: undefined, loading: true, networkStatus: 1, stale: true}
```
And error that 
```js
Error sending the query 'myQuery' Error: Error writing result to store for query
...
Cannot read property 'myQuery' of undefined
```
I didn't work with vue-apollo 2.x, but in [migration guide](https://github.com/Akryum/vue-apollo#migrating-from-vue-apollo-2x-and-apollo-1x) I can see that it supported batching somewhat.

Can you help me with it? Should I pass additional option to vue-apollo? or use another link, or may be this problem will be fixed in 3.0.0-rc?

Thanks

**Comment by aliksend** [comment]
Also I noticed that queries like
```graphql
{ a }
```
and 
```graphql
{ b }
```
didn't combines to one
```graphql
{ a b }
```
but sends to backend like array
```js
[{"operationName":null,"variables":{},"query":"{ a }\n"}, {"operationName":null,"variables":{},"query":"{ b }\n"}]
```
I didn't know about this format of request but my backend successfully process it so I guess that it is valid. I get response like
```json
[{"operationName":null,"payload":{"data":{"a": 1 }}}, {"operationName":null,"payload":{"data":{"b": 2 }}}]
```
But I don't understand why queries isn't combines in simpler way. But probably this question should be in `apollo-link-batch-http` repository.
Also I think that this format of request and response is reason why I got problem and created this issue.
